xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-webjar/src/main/webjar/lightbox.js

```diff
@@ -166,7 +166,9 @@ define('xwiki-lightbox', [
   'blueimp-gallery-fullscreen',
   'blueimp-gallery-indicator'
 ], function($, lightboxDescription, gallery) {
-  var myOpenLightbox;
+  var _openedLightbox;
+  var _slidesData;
+  var _lightboxImages;
```

Comment:
These are internal / private variables of the ``xwiki-lightbox`` so there's no reason to prefix them with underscore. There's no risk of naming conflicts or something like that.